changelog/unreleased/kong/bump-openresty-1.21.4.3.yml

```diff
@@ -0,0 +1,3 @@
+message: "Bumped OpenResty from 1.21.4.2 to 1.21.4.3"
```

Comment:
Keep just one changelog entry of bump from `1.21.4.1` to `1.21.4.3`, not two separate ones.

Reply:
done, thanks